openedx/core/djangoapps/catalog/utils.py

```diff
@@ -109,15 +109,17 @@ def get_program_types(name=None):
         return []
 
 
-def get_programs_with_type(types=None):
+def get_programs_with_type(types=None, include_hidden=True):
```

Comment:
`get_programs()` is called in other contexts (e.g., course dashboard, program dashboard, program certification) to retrieve programs from the cache. I don't fully understand what you're trying to accomplish with "hidden" programs, but they're going to appear in those contexts on edx.org if you merge this as-is. @mjfrey @clintonb is that desired behavior?